css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "oppins", sans-serif;
+  background-image: url("../img/background.png");
+  background-repeat: no-repeat;
+}
+
+.container {
+  margin: 5%;
+  padding-top: 3rem;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1em 1.8em;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 1.2rem;
+  line-height: 1.4rem;
+  text-align: center;
+  color: #6070ff;
+}
+
+.bars {
+  color: #6070ff;
+}
+
+.hero-main-text {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2rem;
+  color: #172b4d;
+}
+
+.hero-minor-text {
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.8rem;
+}
+
+.text-connect {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.2rem;
+  color: #7f8cff;
+}
+
+.icons-hero {
+  color: #505f79;
+  font-size: 0.8rem;
+}
```

Comment:
- [ ] Kindly give space between the **_Let's Connect_** button and the **_social media icons_**. I would suggest you use `12px`. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/221054253-60b56c88-0deb-4622-81cb-3be6db1d5e67.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/221054326-ea1645c5-f1a1-4697-ac6c-d2e1abe21f3a.png)

Reply:
done :heavy_check_mark: